client/components/LoadMessages.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
-import {connect} from 'react-redux'
-import {fetchMessages} from '../actions'
+import { connect } from 'react-redux'
+import { fetchMessages } from '../actions'
```

Comment:
This kind of spacing isn't consistent with the rest of the project. Is this the doing of your linter?